src/API/FeedState.php

```diff
@@ -324,6 +326,36 @@ private function add_feed_registration_state( $result ) {
 		return $result;
 	}
 
+	/**
+	 * Adds to the result variable an array with info about the
+	 * registration and configuration process of the XML feed to the Pinterest API.
+	 *
+	 * @since x.x.x
+	 *
+	 * @param array $result The result array to add values to.
+	 *
+	 * @return array
+	 *
+	 * @throws \Exception PHP Exception.
+	 */
+	private function add_third_party_tags_warning( $result ) {
+
+		$tags = Tracking::get_third_party_installed_tags();
+
+		if ( empty( $tags ) ) {
+			return $result;
+		}
+
+		$result['workflow'][] = array(
+			'label'        => esc_html__( 'Pinterest tag', 'pinterest-for-woocommerce' ),
+			'status'       => 'warning',
+			'status_label' => esc_html__( 'Potential conflicting plugins', 'pinterest-for-woocommerce' ),
+			'extra_info'   => Tracking::get_third_party_tags_warning_message(),
```

Comment:
Instead of separate methods `get_third_party_installed_tags()` and `get_third_party_tags_warning_message()`, it might make more sense to have only one method, probably the warning message method. If there are any warning messages, then we can display them, but if they are empty then we know not to send any warnings.